canvas_modules/common-canvas/src/common-properties/components/editor-form/editor-form.jsx

```diff
@@ -89,6 +89,18 @@ class EditorForm extends React.Component {
 		return true;
 	}
 
+	componentDidUpdate(prevProps) {
+		// Scroll to the selected accordion even when clicked from Alerts tab
+		if (this.scrollToAlert || prevProps.activeTab !== this.props.activeTab) {
+			const activeTabId = this.props.activeTab;
+			const activeTabElement = document.getElementsByClassName(`${activeTabId}`);
+			if (activeTabId && activeTabElement.length > 0) {
+				activeTabElement[0].scrollIntoView({ behavior: "smooth" });
+			}
+			this.scrollToAlert = false;
```

Comment:
Why do you need `this.scrollToAlert`?  Shouldn't `prevProps.activeTab !== this.props.activeTab` be enough?

Reply:
When I click on alert link then for the first time it is set as activeTab and if I scroll up to Alerts tab again and click then this check fails so I included `this.scrollToAlert`